content/common/cli-reference.md

```diff
@@ -2319,4 +3378,21 @@ CLI commands have four phases that indicate levels of stability:
 
 {{ blockEnd }}
 
+{{ startTab "v1.2.0"}}
+
+| Command                                                                               | Stability    |
+| ------------------------------------------------------------------------------------- | ------------ |
+| <code>spin add</code>                                                                 | Stable       |
+| <code>spin build</code>                                                               | Stable       |
+| <code>spin new</code>                                                                 | Stable       |
+| <code>spin plugins <install&vert;list&vert;uninstall&vert;update&vert;upgrade></code> | Stable       |
+| <code>spin templates <install&vert;list&vert;uninstall&vert;upgrade></code>           | Stable       |
+| <code>spin up</code>                                                                  | Stable       |
+| <code>spin cloud <deploy&vert;login></code>                                           | Stabilizing  |
+| <code>spin registry</code>                                                            | Stabilizing  |
+| <code>spin watch</code>                                                               | Experimental |
```

Comment:
Probably stabilizing

Reply:
Thanks @calebschoepp  - new commit has this `watch` feature updated from Experimental to Stabalizing.